### PR TITLE
Fixes Plasma Fist fighting style

### LINKED ISSUE #10617: Plasma Fist Scroll no longer informs the user of combos
1. Use plasma fist scroll

2. Learn nothing

### PR DESCRIPTION
**Pull request by LanCartwright**
how about a knuckle sandwitch

also i did this with a bunch of slavs 

fixes #10617

### CODE REVIEW

**Comment by LanCartwright** [comment]
https://github.com/tgstation/-tg-station/issues/10617 fixes this

**Comment by KorPhaeron** [comment]
Can you do this for the wrestling style as well?

**Comment by LanCartwright** [comment]
added wresting fighting style. suprisingly there's a lack of information on it.

**Comment by duncathan** [comment]
This is bad
Martial arts are datums, right? Make the instructions a proc or var on the datum, then make the verb just call the proc of the art you choose from those you have.

**Comment by tkdrg** [comment]
Even better if you make the verb generate the instructions from the datums, thus making it easier to expand

File: code/datums/martial.dm [hunk 1]
```diff
@@ -175,12 +186,24 @@
 	D.apply_damage(10, STAMINA, affecting, armor_block)
 	return 1
 
+ /mob/living/carbon/human/proc/wrestling_help()
```
Comment (tkdrg) [inline code/datums/martial.dm]: Remove the space from the beginning of this line.

**Comment by duncathan** [comment]
You haven't listened to a word I've said @LanCartwright
Pay me ten bucks and I'll make the code good for you

**Comment by duncathan** [comment]
Hell I'll even settle for five if you want

**Pull request merged**